Assets/MixedRealityToolkit.SDK/Features/Utilities/StabilizationPlaneModifier.cs

```diff
@@ -0,0 +1,390 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using UnityEngine.XR.WSA;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
+{
+    [Serializable]
+    public struct StabilizationPlaneOverride
```

Comment:
>StabilizationPlaneOverride [](start = 18, length = 26)

Nit: Please add a doccomment for this class and its fields.

Reply:
We use these comments to generate API documentation. Agree they should be added.

Thanks!